### PR TITLE
Components

### PR DESCRIPTION
**Pull request by OpondoG**
# In this pull request I:
## Initialize the Bookstore project with components
- [x] The building blocks of your app should be set as reusable components. Create a directory for your components.
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- [x] Books - the default view:
- [x] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- [x] Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- [x] Should display a form for adding a book (no functionality yet).
- [x] Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
- [x] Categories:
- [x] Should display a button "Check status" only.
- [x] Styling is not required at this point.
- [x] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Make sure that you used the correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (codepantha) [inline README.md:64]: - [ ] The link to the MIT license is broken. Please fix it by creating a LICENSE file 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @OpondoG 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Initialized project with components
✔️ No linter errors
✔️ Used GitFlow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+This project is [MIT](./MIT.md) licensed.
```
Comment (codepantha) [inline README.md]: - [x] Well done for attempting to fix the issue with your license though I see you are having some difficulties.
Kindly note that your license file is name `LICENSE` but you are linking to `./MIT.md` and that's where the problem lies. Please replace the highlighted line with: This project is `[MIT](./LICENSE)` licensed 👍 😎

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @OpondoG 👋, 

Well done for attempting to implement the required changes 👏 
You are almost there 💯



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (OpondoG) [reply README.md:62]: I've tried so but the link is still breaking

**Review by danielufeli — APPROVED** [review]
Hi @OpondoG ,

Your, project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**